modules/mobile/app/serializers/mobile/v0/letters_serializer.rb

```diff
@@ -9,16 +9,37 @@ class LettersSerializer
       attributes :letters
 
       def initialize(id, letters, options = {})
-        letters.map! do |letter|
-          letter.name = 'Benefit Summary and Service Verification Letter' if letter.letter_type == 'benefit_summary'
-          if letter.letter_type == 'benefit_summary_dependent'
-            letter.name = 'Dependent Benefit Summary and Service Verification Letter'
+        if Flipper.enabled?(:mobile_lighthouse_letters, @current_user)
```

Comment:
This isn't pretty but as soon as the lighthouse service gets put on production then we can remove all this EVSS code.